src/main/kotlin/notary/EthWalletsProviderIrohaImpl.kt

```diff
@@ -13,15 +13,15 @@ import sidechain.iroha.util.getRelays
  * @param irohaConfig - Iroha configuration
  * @param keypair - Iroha keypair to query
  * @param irohaNetwork - iroha network layer
- * @param account - account that contains details
- * @param setter - account that has set details
+ * @param notaryIrohaAccount - notaryIrohaAccount that contains details
+ * @param registrationAccount - registrationAccount that has set details
  */
 class EthWalletsProviderIrohaImpl(
-    val irohaConfig: IrohaConfig,
-    val keypair: Keypair,
-    val irohaNetwork: IrohaNetwork,
-    val account: String,
-    val setter: String
+    private val irohaConfig: IrohaConfig,
+    private val keypair: Keypair,
+    private val irohaNetwork: IrohaNetwork,
+    private val notaryIrohaAccount: String,
+    private val registrationAccount: String
```

Comment:
In general case, account details can be set not by `registrationAccount`. I think we need more meaningful name, like 'accountDetailSetter' and 'accountDetailHolder'.

Reply:
yes that is true. but this is wallets provider class and the only account that can add new wallet to Iroha is registrationAccount.